app/services/service_health/monitor.py

```
@@ -0,0 +1,440 @@
+    pattern_map = _LOG_PATTERN_MAP
+
+    def sink(message):
+        record = message.record
+        if record["level"].no < 30:  # WARNING = 30, skip DEBUG/INFO
+            return
+        text = record["message"].lower()
+        try:
+            loop = asyncio.get_running_loop()
+            for rule, phrases in pattern_map.items():
+                if any(phrase in text for phrase in phrases):
+                    loop.create_task(service_health_monitor.record_error(rule))
```

Ruff (0.15.10)

[warning] Store a reference to the return value of `loop.create_task`

(RUF006)